### ISSUE TITLE
password from keyboard and by touch are not same

### ISSUE THREAD

**Issue by EngMFKS** [ChromisPos/ChromisPOS#42]
password from keyboard and by touch are not same

**Comment by Geo-Ron** [comment]
Yes it is, you only need to change the touch input type by pressing the dot sign

![image](https://user-images.githubusercontent.com/23144316/39986306-965f5bc0-5761-11e8-92a0-1ad2334d00a4.png)
